sporkfish/endgame_tablebases/lila_endgame_tablebase.py

```diff
@@ -23,9 +28,9 @@ def query_bestmove(board_fen: str) -> Optional[chess.Move]:
 
         :raises ConnectionError: If there is an issue connecting to the tablebase service.
         """
-        full_url = LilaTablebase.base_url + board_fen
+        FULL_URL = LilaTablebase.base_url + board_fen
```

Comment:
Think base_url should be BASE_URL. Tiny nitpick, I think in the style guide only constants should have caps so full_url was probably originally correct